VideoWeb/VideoWeb/ClientApp/src/app/services/stream-mixer.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class StreamMixerService {
+    mergeAudioStreams(...streams: MediaStream[]) {
+        const audioContext = new AudioContext();
+        const destination = audioContext.createMediaStreamDestination();
+
+        const streamsWithAudio = streams.filter(s => s.getAudioTracks().length > 0);
+        streamsWithAudio.forEach(s => {
+            const stream = new MediaStream();
+            stream.addTrack(s.getAudioTracks()[0]);
```

Comment:
Do you not want to mix all tracks from  all streams?

Reply:
Sorry I meant to say all Audio tracks. Not just the first track.

Reply:
Also to test this class requires a lot of mocks it's not worth the effort since it's not representative of the actual behaviour of MediaStream and AudioContext. It would be tests for the sake of coverage and nothing more